Merge tag '0.15' into develop

juandent opened the pull request:
Will adapt sources for enabling use with Microsoft Visual Studio 2015 Update 2 compiler...

rbock committed:
Merge tag '0.15' into develop

Maintenance release

rbock commented:
This pull request is not ready yet, right?

I am just asking because as of now, I see no changes.

juandent commented:
Sorry had a problem with my laptop
Please allow me some time

Regards
Juan

Sent from my iPhone

> On Apr 19, 2016, at 10:53 PM, Roland Bock notifications@github.com wrote:
> 
> This pull request is not ready yet, right?
> 
> I am just asking because as of now, I see no changes.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub

rbock commented:
Sure :-)

rbock committed:
Remove compile error

rbock committed:
Added travis CI config

rbock committed:
Adjusted date lookup like in sqlpp11

rbock committed:
Fixed several bugs in test programs

rbock committed:
Fixed build stuff

- corrected includes for mysql
- find paths in cmake
- handle missing thread_local support in libc++

rbock committed:
Activated all travis CI configurations
